helm/azure-admission-controller/templates/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "labels.common" . | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
Why 3 ? Isn't 2 enough with this PDB?

Reply:
That was the consent because of the amount of masters running in production, we can also 2 replicas it should be enough 👍🏻.